monkey/infection_monkey/credential_collectors/i_credential_collector.py

```diff
@@ -1,10 +1,10 @@
 from abc import ABC, abstractmethod
-from typing import List
+from typing import List, Mapping, Union
 
 from .credentials import Credentials
 
 
 class ICredentialCollector(ABC):
     @abstractmethod
-    def collect_credentials(self) -> List[Credentials]:
+    def collect_credentials(self, options: Union[Mapping, None]) -> List[Credentials]:
```

Comment:
```suggestion
    def collect_credentials(self, options: Optional[Mapping]) -> List[Credentials]:
```

Reply:
Totally forgot that this thing exists :D